### PR TITLE
fix NPE if invalid http was sent (empty headers)

### PR DESCRIPTION
**Pull request by seut**
also change log-level from error to debug for
unhandled exception due to invalid http requests

### CODE REVIEW

**Comment by seut** [comment]
@chaudum pls review

File: blob/src/main/java/io/crate/http/netty/HttpBlobHandler.java [hunk 1]
```diff
@@ -258,7 +258,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e)
             body = ex.getMessage();
         } else {
             status = HttpResponseStatus.INTERNAL_SERVER_ERROR;
-            LOGGER.error("unhandled exception:", ex);
+            LOGGER.debug("unhandled exception:", ex);
```
Comment (dobe) [inline blob/src/main/java/io/crate/http/netty/HttpBlobHandler.java]: s/debug/warn/

Comment (seut) [inline blob/src/main/java/io/crate/http/netty/HttpBlobHandler.java]: with warn it will be logged by default. do we want this for sure?

Comment (cleishm) [inline blob/src/main/java/io/crate/http/netty/HttpBlobHandler.java]: We should send back a "you sent me something invalid" in response to the request, but I'd propose to keep it out of the log.

**Pull request merged**